packages/example/package.json

```diff
@@ -12,7 +12,7 @@
     "react-dom": "^18.0.0"
   },
   "peerDependencies": {
-    "wagmi": "^0.3.0"
+    "wagmi": "^0.3.5"
```

Comment:
Since RainbowKit doesn't have any changes and the latest version of wagmi is still satisfied by our version range, we should probably leave this off. What do you think?

Reply:
Sorry, this comment is meant to be for this line: https://github.com/rainbow-me/rainbowkit/pull/368/files#diff-ab652f6c376cf926bae2d5acb6eaeae51dfc8c6dc57dfd21eb0d3e98fc1001cdR45

Reply:
Updated